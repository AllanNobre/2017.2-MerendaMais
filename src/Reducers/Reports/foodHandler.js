const foodHandler = [
  { label: 'Uniforme completo', markedYes: false, markedNo: false, status: false, key: 0 },
  { label: 'Manipuladoras com maquiagem (batom, sombra, lápis de olho, etc)', markedYes: false, markedNo: false, status: false, key: 1 },
  { label: 'Manipuladoras com esmalte nas unhas', markedYes: false, markedNo: false, status: false, key: 2 },
  { label: 'Manipuladores com unhas compridas', markedYes: false, markedNo: false, status: false, key: 3 },
  { label: 'Manipuladores doentes', markedYes: false, markedNo: false, status: false, key: 4 },
  { label: 'Manipuladores sem touca', markedYes: false, markedNo: false, status: false, key: 5 },
  { label: 'Manipuladoras usando brincos, anéis ou pulseiras', markedYes: false, markedNo: false, status: false, key: 6 },
  { label: 'Visitantes/Manipuladores sem touca', markedYes: false, markedNo: false, status: false, key: 7 },
];

export default foodHandler;
