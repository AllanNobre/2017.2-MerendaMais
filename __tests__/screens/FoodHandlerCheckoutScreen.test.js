import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// imported as a connected component!
import FoodHandlerCheckoutContainer from '../../src/Containers/FoodHandlerCheckoutContainer';
import foodHandler from '../../src/Reducers/Reports/foodHandler';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {
  report: {
    foodHandler,
    foodHandlerObservation: '',
  },
};

const store = mockStore(initialState);

describe('Testing FoodHandlerCheckout Screen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <FoodHandlerCheckoutContainer />,
      { context: { store } },
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
