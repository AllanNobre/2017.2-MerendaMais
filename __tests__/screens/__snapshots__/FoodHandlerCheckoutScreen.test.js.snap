// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FoodHandlerCheckout Screen renders as expected 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <Header
    backButton={true}
    subTitle="Manipuladores de Alimentos"
    title="Relatório"
  />
  <KeyboardAvoidingView
    behavior="padding"
    keyboardVerticalOffset={0}
    style={
      Object {
        "flex": 6,
        "flexDirection": "column",
        "marginBottom": 9,
      }
    }
  >
    <ScrollViewMock>
      <View>
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "fontWeight": "bold",
                "paddingLeft": 10,
                "paddingTop": 5,
              }
            }
          >
            Sim
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "fontWeight": "bold",
                "paddingLeft": 20,
                "paddingTop": 5,
              }
            }
          >
            Nao
          </Text>
        </View>
        <View
          key="0"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Uniforme completo
          </Text>
        </View>
        <View
          key="1"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladoras com maquiagem (batom, sombra, lápis de olho, etc)
          </Text>
        </View>
        <View
          key="2"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladoras com esmalte nas unhas
          </Text>
        </View>
        <View
          key="3"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladores com unhas compridas
          </Text>
        </View>
        <View
          key="4"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladores doentes
          </Text>
        </View>
        <View
          key="5"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladores sem touca
          </Text>
        </View>
        <View
          key="6"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Manipuladoras usando brincos, anéis ou pulseiras
          </Text>
        </View>
        <View
          key="7"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#008000"
            />
          </View>
          <View>
            <Checkbox
              checkboxStyle={
                Object {
                  "borderColor": "black",
                  "borderRadius": 2,
                  "borderWidth": 2,
                  "flexWrap": "wrap",
                  "height": 25,
                  "marginLeft": 10,
                  "marginRight": 10,
                  "marginTop": 10,
                  "width": 25,
                }
              }
              defaultColor="#fff"
              disabled={false}
              disabledColor={null}
              onSelect={[Function]}
              selected={false}
              selectedColor="#B22222"
            />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "flex": 1,
                "fontSize": 15,
                "paddingTop": 15,
              }
            }
          >
            Visitantes/Manipuladores sem touca
          </Text>
        </View>
      </View>
      <View
        behavior="padding"
      >
        <View
          style={
            Object {
              "paddingLeft": 10,
              "paddingRight": 10,
              "paddingTop": 30,
            }
          }
        >
          <TextInput
            multiline={true}
            onChangeText={[Function]}
            placeholder="Observações (opcional)"
            style={
              Object {
                "borderRadius": 12,
                "borderWidth": 1,
                "fontSize": 37.5,
                "height": 333.5,
                "paddingLeft": 10,
                "paddingTop": 10,
                "textAlignVertical": "top",
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
      </View>
      <TouchableOpacity
        activeOpacity={0.2}
        key="setObservation"
        onPress={[Function]}
        style={
          Object {
            "backgroundColor": "#FF9500",
            "borderRadius": 7,
            "borderWidth": 1,
            "justifyContent": "flex-end",
            "marginBottom": 20,
            "marginHorizontal": 15,
            "marginTop": 30,
            "paddingVertical": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFF",
              "textAlign": "center",
            }
          }
        >
          Concluir
        </Text>
      </TouchableOpacity>
    </ScrollViewMock>
  </KeyboardAvoidingView>
</View>
`;
